src/plugins/intel_gpu/src/kernel_selector/core/common/primitive_db.h

```diff
@@ -22,7 +22,7 @@ struct primitive_db {
     primitive_db();
 
     std::vector<code> get(const primitive_id& id) const;
-    std::vector<code> get_batch_header_str() const { return std::move(batch_header_str); }
+    std::vector<code> get_batch_header_str() const { return batch_header_str; }
```

Comment:
why did you fixed this?

Reply:
In this case, std::move is useless, as this method is marked as const. So probably, we should remove const or remove std::move. But still, this is not the scope of this PR, I have reverted it.